test/e2e/lua/dynamic_configuration.go

```diff
@@ -88,6 +86,7 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 			replicas := 2
 			err := framework.UpdateDeployment(f.KubeClientSet, f.IngressController.Namespace, "http-svc", replicas, nil)
 			Expect(err).NotTo(HaveOccurred())
+			time.Sleep(5 * time.Second)
```

Comment:
we need to give some time for the logs to appear, 5 seconds might be a bit too conservative but we can revisit this later